wger/urls.py

```diff
@@ -98,6 +98,7 @@
 
 # Core app
 router.register(r'userprofile', core_api_views.UserProfileViewSet, base_name='userprofile')
+router.register(r'user/registration', core_api_views.UserViewSet, base_name='user')
```

Comment:
we can have the url pattern be the same as the others. also the `base_name` doesn't imply registration.